src/main/java/edu/ie3/datamodel/io/source/influxdb/InfluxDbWeatherSource.java

```diff
@@ -152,24 +156,25 @@ public InfluxDbWeatherSource(
     return measurementsMap.get(MEASUREMENT_NAME_WEATHER).stream()
```

Comment:
something weird is going on in the docs of the function of `optTimeBasedValueStrem` on line 149 - would be nice if we can fix that here as well